src/java/edu/slu/tpen/servlet/CanvasServlet.java

```diff
@@ -49,7 +52,7 @@ public class CanvasServlet extends HttpServlet{
      */
```

Comment:
Arrghh the old documentation was wrong.  Can you update this bit?
"serialisation of the requested T-PEN **canvas**."